packages/store/src/state-factory.ts

```diff
@@ -2,18 +2,40 @@ import { Injector, Injectable, SkipSelf, Optional } from '@angular/core';
 import { Observable, of, forkJoin, from } from 'rxjs';
 import { shareReplay, takeUntil, map } from 'rxjs/operators';
 
-import { META_KEY, StateContext, ActionOptions, NgxsLifeCycle } from './symbols';
-import { topologicalSort, buildGraph, findFullParentPath, nameToState, MetaDataModel, isObject } from './internals';
+import { META_KEY, StateContext, NgxsLifeCycle } from './symbols';
+import {
+  topologicalSort,
+  buildGraph,
+  findFullParentPath,
+  nameToState,
+  isObject,
+  ActionHandlerMetaData,
+  ObjectKeyMap,
+  StateKlass,
+  GetStateFn,
+  SetStateFn,
+  DispatchFn
+} from './internals';
 import { getActionTypeFromInstance, setValue, getValue } from './utils';
 import { ofActionDispatched } from './of-action';
+import { Actions } from '@ngxs/store';
+import { InternalActions } from '@ngxs/store/src/actions-stream';
```

Comment:
This import looks wrong.

Reply:
Both of those should import directly not use the alias

Reply:
I think this is a "feature" of autoimports in vscode and intellij.
the editor seems to sometimes want to do a absolute import.

Just do a last check before committing, making sure that there are no imports to /src/.

Maybe we can add a tslint rule that forbids importing from `@ngxs/store/src/*` ?

Reply:
that might not be a bad idea

Reply:
but even then we shouldn't import from @ngxs/store when we are inside of the store

Reply:
Agreed